frontend/src/app/apps/_components/App/App.tsx

```diff
@@ -1,13 +1,18 @@
 import styles from './App.module.css';
 
+interface InformationDetails {
+	title: string;
+	description: string;
+	owners: string;
+}
+
 interface AppInformation {
-	information: string[];
+	appInformation: InformationDetails;
 }
 
-const App: React.FunctionComponent<AppInformation> = ({
-	information,
+const App = ({
+	appInformation: { title, description, owners },
```

Comment:
Do we need to restate that this is the App Component's `appInformation`? Could we reduce this title without affecting clarity?